pkg/odo/cli/component/describe.go

```diff
@@ -48,12 +48,12 @@ func (do *DescribeOptions) Validate() (err error) {
 		return odoutil.ThrowContextError()
 	}
 
-	isExists, err := component.Exists(do.Context.Client, do.componentName, do.Context.Application)
+	existsInCluster, err := component.Exists(do.Context.Client, do.componentName, do.Context.Application)
 	if err != nil {
 		return err
 	}
-	if !isExists {
-		return fmt.Errorf("component %s does not exist", do.componentName)
+	if !existsInCluster {
+		return fmt.Errorf("component %s not pushed to OpenShift cluster, use `odo push` to deploy the component", do.componentName)
```

Comment:
:+1:

Reply:
@cdrage Maybe `hasn't been pushed to a OpenShift cluster`?

Reply:
Also `a openshift cluster` or maybe `any openshift cluster`?

Reply:
"any" or "hasn't" doesn't make sense :( I am going by the issue: https://github.com/openshift/odo/issues/1847

Reply:
@kadel @jorgemoralespou  TBH the grammar seems out of place to me. Also according to https://github.com/openshift/odo/issues/1847#issuecomment-505934386, `the openhsift` is missing :)